app/_hub/kong-inc/application-registration/how-to/_index.md

```diff
@@ -36,6 +36,64 @@ curl -X PATCH http://localhost:8001/plugins/<plugin_id> \
   --data "config.auto_approve=true"
 ```
 
+### Enable proxy with consumer's credential
+
+You can enable `enable_proxy_with_consumer_credential`. It will introduce
+an additional method for accessing the service using the consumer's credential.
+
+```
+curl -X POST http://localhost:8001/services/<service>/plugins \
+    --data "name=application-registration"  \
+    --data "config.display_name=<my_service_display_name>" \
+    --data "config.enable_proxy_with_consumer_credential=true"
+```
+
+Or, update your current configuration with a `PATCH` request.
+Replace `<plugin_id>` with the `id` of your plugin.
+
+```
+curl -X PATCH http://localhost:8001/plugins/<plugin_id> \
+  --data "config.enable_proxy_with_consumer_credential=true"
+```
+
+And then, create a plugin `key-auth` and route for the service.
+
+```
+curl -X POST http://localhost:8001/services/<service>/routes \
+    --data "path=/test"
+```
+
+```
+curl -X POST http://localhost:8001/services/<service>/plugins \
+    --data "name=key-auth" \
+    --data "config.key_names=apikey"
+```
+
+Create a consumer and a consumer's credential for the `key-auth`.
+
+```
+curl -X POST http://localhost:8001/consumers \
+    --data "username=test"
+```
+
+```
+curl -X POST http://localhost:8001/consumers/<consumer_id>/key-auth \
+    --data "key:<apikey>"
+```
+
+Request the route for the service, then the request will be return 200
+else return 401 if disabling `enable_proxy_with_consumer_credential`.
```

Comment:
When `enable_proxy_with_consumer_credential` is enabled, constructing a request to the service should yield a 200 response, rather than the 401 response when disabled.